src/infra/tokenizer/token-manager.ts

```diff
@@ -0,0 +1,52 @@
+import * as E from "fp-ts/lib/Either";
+import * as TE from "fp-ts/lib/TaskEither";
+import { pipe } from "fp-ts/lib/function";
+
+import { FiscalCode } from "@pagopa/io-functions-services-sdk/FiscalCode";
+import { config } from "../../app/config";
+import { tokenizerApiClient } from "./client";
+import { SignerId } from "../../signer/signer";
+
+export const getSignerIdFromFiscalCode = (fiscalCode: FiscalCode) =>
```

Comment:
I suggest you to rename these methods to `tokenizeFiscalCode` and `getFiscalCodeFromToken` or something similar since, in my opinion, they shouldn't relate to "signers."